ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh

```diff
@@ -38,7 +38,7 @@ timeout -s 9 15m ssh $SSHOPTS root@$IP bash - << EOF |& sed -e 's/.*auths.*/***
 cd dev-scripts
 
 # Get install-gather, if there is one
-cp log-bundle\* /tmp/artifacts || true
+cp /root/dev-scripts/ocp/ostest/log-bundle\* /tmp/artifacts || true
```

Comment:
It would be better in future to avoid hard-coding this path to avoid breaks in case the default cluster name is changed (ie devscript-setup step could export CLUSTER_NAME in a shared file)

Reply:
Yea it's a good idea but the ostest is also hardcoded in at least the kubeconfig var.

Should be easy enough to fix if we end up needing to do multiple clusters on the same box or change the name for some reason.

Reply:
Isn't KUBECONFIG built based on CLUSTER_NAME?

Reply:
Was referring to this: https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L93

Reply:
Right also that one should be refactored